.eleventy.js

```diff
@@ -51,6 +51,10 @@ module.exports = (config) => {
   })
   config.setLibrary('njk', nunjucksEnv)
 
+  config.addNunjucksGlobal('flags', {
```

Comment:
I tried to add this to the vars first, but those aren't available in macros so have added this as a global instead.